### DISCUSSION TITLE
Feature Request: Add "addresses" as keyword to link pull request to issue

### DISCUSSION THREAD

**Discussion by gkulin** [community/community#38508] [Pull Requests]
Suggestion: Add "addresses" as a keyword to link a pull request to an issue

Why: It's a semi-common alternative to existing keywords (see https://github.com/github/docs/pull/20362, https://github.com/github/docs/pull/13682, https://github.com/github/docs/pull/7683 for examples in the GitHub Docs repo alone). People who might not know that they can use keywords to link issues might use "addresses" because it is more natural to them than the other keywords, so making it a keyword will provide a better experience to folks who were previously unaware of the feature

Related Doc: https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword

**Comment by ossanna16** [comment]
Hi there @gkulin and welcome to our community! Thank you for this feature request!

**Comment by designatednerd** [comment]
Upvoting this after having four PRs with `addresses` merged that didn't get picked up because I didn't realize it wasn't a keyword 🤦‍♀️
